### ISSUE TITLE
alarm_notifier configuration File

### ISSUE THREAD

**Issue by vgeannin** [nokia/OSSMediator#46]
Hi All,

What should we put in this file if we want to be notify for all alarms ?
Does this file is enough : 

> webhook_url: http://localhost:8090/
> alarm_sync_duration: 30
> group_events: false
> notify_clear_event: true
> message_format: json

Thanks

**Comment by soursing** [comment]
@vgeannin Currently alarm notifier only supports alerting for specific alarms specified using below configuration.  
```
  radio_alarm_filters:
    - specific_problem: <ALARM SPECIFIC PROBLEM>
    - specific_problem: <ALARM SPECIFIC PROBLEM>
      fault_ids:
        - <FAULT ID>
        - <FAULT ID>
  dac_alarm_filters:
    - alarm_id: <ALARM ID>
  core_alarm_filters:
    - alarm_id: <ALARM ID>
```

We'll add support to enable all alarm notification in next release.

**Comment by soursing** [comment]
Hi @vgeannin,

We've released `4.2` version that supports wildcard to enable notification for all alarms.
Please refer https://github.com/nokia/OSSMediator/blob/master/OSSMediatorCollector/README.md#alarm-notification for more detail.

Example configuration to enable notification for all alarms:
``` 
 radio_alarm_filters:
    - specific_problem: *
  dac_alarm_filters:
    - alarm_id: *
  core_alarm_filters:
    - alarm_id: *
  ```

**Comment by soursing** [comment]
Closing issue as no response received.
